multinode/storage/service.go

```diff
@@ -126,6 +126,10 @@ func (service *Service) TotalUsageSatellite(ctx context.Context, satelliteID sto
 	cache := make(UsageStampDailyCache)
 
 	for _, node := range nodesList {
+		nodeinfo := service.nodes.FetchNodeInfo(ctx, node)
```

Comment:
The same that I commented before, we would also use here the function that checks the status of the node.